dbms/src/Common/FailPoint.cpp

```diff
@@ -39,12 +39,14 @@ namespace DB
     M(exception_before_dmfile_remove_from_disk)                   \
     M(force_triggle_background_merge_delta)                       \
     M(force_triggle_foreground_flush)                             \
+    M(exception_before_mpp_make_non_root_mpp_task_public)         \
```

Comment:
registered and visible make me a little confused..
Or use registered and active?

Reply:
Done